import_export_extensions/templates/admin/import_export_extensions/celery_export_status.html

```diff
@@ -42,7 +42,7 @@
         <div class="form-row">
           <label>Progress</label>
           <p id="progress_bar_html">
-            <progress max="100" id="progress-bar" data-url="{{ export_job_url }}"></progress>
+            <progress max="100" id="progress-bar" data-url="{{ export_job_url }}" data-label="Waiting..."></progress>
```

Comment:
Would be it be an idea to put 'Waiting...' in a [translate](https://docs.djangoproject.com/en/5.1/topics/i18n/translation/#translate-template-tag) tag?